<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Youngs-MacBook-Pro.local" name="main.java.com.dbyl.tests.Test" tests="1" failures="0" timestamp="27 Mar 2017 14:18:22 GMT" time="40.615" errors="1">
  <testcase name="myTest" time="40.615" classname="main.java.com.dbyl.tests.Test">
    <error type="java.lang.NullPointerException" message="while trying to invoke the method main.java.com.dbyl.libarary.utils.Locator.getElement() of a null object loaded from local variable &#039;locator&#039;">
      <![CDATA[java.lang.NullPointerException: while trying to invoke the method main.java.com.dbyl.libarary.utils.Locator.getElement() of a null object loaded from local variable 'locator'
	at main.java.com.dbyl.libarary.utils.BasePage.findElement(BasePage.java:281)
	at main.java.com.dbyl.libarary.utils.BasePage$1.apply(BasePage.java:245)
	at main.java.com.dbyl.libarary.utils.BasePage$1.apply(BasePage.java:1)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:209)
	at main.java.com.dbyl.libarary.utils.BasePage.isElementPresent(BasePage.java:241)
	at main.java.com.dbyl.libarary.utils.BasePage.isElementPresent(BasePage.java:267)
	at main.java.com.dbyl.tests.Test.myTest(Test.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- myTest -->
</testsuite> <!-- main.java.com.dbyl.tests.Test -->
